bin/hdfcoinc/pycbc_page_snrifar

```diff
@@ -38,41 +46,96 @@ parser.add_argument('--version', action='version', version=pycbc.version.git_ver
+                         'foreground triggers to plot if there were any '
+                         'hierarchical removals done. Choosing None plots '
+                         'the inclusive backgrounds prior to any '
+                         'hierarchical removals with the updated FARs for '
+                         'foreground triggers after hierarchical removal(s). '
+                         'Choosing 0 means plotting inclusive background '
+                         'from prior to any hierarchical removals with FARs '
+                         'for foreground triggers prior to hierarchical '
+                         'removal. Choosing 1 means plotting the inclusive '
+                         'background after doing 1 hierarchical removal, and '
+                         'includes updated FARs from after 1 hierarchical '
+                         'removal. [default=None]')
+parser.add_argument('--fg-marker', default='^',
+                    help='Marker to use for the foreground triggers, '
+                         '[default = ^]')
+parser.add_argument('--fg-marker-h-rm', default='v',
+                    help='Marker to use for the hierarchically removed '
+                         'foreground triggers. [default = v]')
 parser.add_argument('--closed-box', action='store_true',
-      help="Make a closed box version that excludes foreground triggers")
-parser.add_argument('--xmin', type=float, help='Set the minimum value of the x-axis')
-parser.add_argument('--xmax', type=float, help='Set the maximum value of the x-axis')
-parser.add_argument('--ymin', type=float, help='Set the minimum value of the y-axis (in units of 1/years)')
-parser.add_argument('--ymax', type=float, help='Set the maximum value of the y-axis (in units of 1/years)')
+                    help="Make a closed box version that excludes foreground "
+                         "triggers")
+parser.add_argument('--xmin', type=float,
+                    help='Set the minimum value of the x-axis')
+parser.add_argument('--xmax', type=float,
+                    help='Set the maximum value of the x-axis')
+parser.add_argument('--ymin', type=float,
+                    help='Set the minimum value of the y-axis ' 
+                         '(in units of 1/years)')
+parser.add_argument('--ymax', type=float,
+                    help='Set the maximum value of the y-axis '
+                         '(in units of 1/years)')
 args = parser.parse_args()
 
-args.cumulative = not args.not_cumulative
-
 if args.verbose:
     log_level = logging.INFO
     logging.basicConfig(format='%(asctime)s : %(message)s', level=log_level)
-    
+
 logging.info('Read in the data')
 f = h5py.File(args.trigger_file, 'r')
 
+# Parse which inclusive background to use for the plotting
+h_inc_back_num = args.h_iter_inc_background
+
+try:
+    h_iterations = f.attrs['hierarchical_removal_iterations']
+except KeyError:
+    h_iterations = 0
+
+if h_inc_back_num is None:
+    h_inc_back_num = None
```

Comment:
This doesn't make any sense. Did you mean to set `h_inc_back_num = h_iterations` in this case?